venia-integration-tests/cypress.config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "https://develop.pwa-venia.com/",
+    "baseUrl": "http://0.0.0.0:10000/",
```

Comment:
this will break the CI

Reply:
Oops, you're right. Thought this file was in the `.gitIgnore` file.